bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/ThingStatusDetail.java

```diff
@@ -23,7 +23,8 @@
     CONFIGURATION_ERROR(7),
     BRIDGE_OFFLINE(8),
     FIRMWARE_UPDATING(9),
-    DUTY_CYCLE(10);
+    FIRMWARE_UPDATE_PENDING(10),
+    DUTY_CYCLE(11);
```

Comment:
This changes the id of the DUTY_CYCLE enum. As we do not persist Thing statuses, I assume that we can ignore this, do you agree?